components/Buttons/src/MDCFloatingButton+Animation.h

```diff
@@ -0,0 +1,37 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCFloatingButton.h"
+
+@interface MDCFloatingButton (Animation)
+
+/**
+ Expand this button to its unscaled (normal) size.
+
+ @param animated YES if the size change should be animated.
+ @param completion a completion block to call after the size change is complete.
+ */
+- (void)expand:(BOOL)animated completion:(void (^_Nullable)(void))completion;
+
```

Comment:
As discussed, please add docs about how this manipulates the transform and include a pointer to Apple's relevant docs.